arquillian-recorder-video-base/arquillian-recorder-video-api/src/main/java/org/arquillian/extension/recorder/video/VideoConfiguration.java

```diff
@@ -156,7 +156,7 @@ private void validate(ReporterConfiguration reporterConfiguration) {
         } catch (IllegalArgumentException ex) {
             throw new VideoConfigurationException(
                 "Video type you specified in arquillian.xml is not valid video type."
-                    + "Supported video types are: " + VideoType.getAll());
+                    + "Supported video types are: " + VideoType.getAll(), ex);
```

Comment:
remove that ex from here and include the video type a user has configured and is not supported into the exception message, constructed by String.format